src/ziggurat/streams.clj

```diff
@@ -42,12 +42,10 @@
     (.put StreamsConfig/REPLICATION_FACTOR_CONFIG (int changelog-topic-replication-factor))
     (.put ConsumerConfig/AUTO_OFFSET_RESET_CONFIG auto-offset-reset-config)))
 
-(defn- get-metric-namespace [default topic]
-  (str (name topic) "." default))
-
 (defn- log-and-report-metrics [topic-entity message]
-  (let [message-read-metric-namespace (get-metric-namespace "message" (name topic-entity))]
-    (metrics/increment-count message-read-metric-namespace "read"))
+  (let [topic-entity-name             (name topic-entity)
+        message-read-metric-namespace "message"]
+    (metrics/increment-count message-read-metric-namespace "read" topic-entity-name))
```

Comment:
we're changing the metric-name here, this will not be backward compatible.

Reply:
done, for every call to (increment-count) added another one using the new metric-namesapce